spec/fabricators/user_fabricator.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Fabricator(:user) do
+  email { FFaker::Internet.email }
```

Comment:
```suggestion
  email { FFaker::Internet.unique.email }
```

This will avoid getting duplicate email if in a test you need to Fabricate multiple users 👍